index.html

```diff
@@ -4,20 +4,37 @@
 	<meta charset="UTF-8">
 	<link rel="stylesheet" href="style.css">
 	<link href="https://fonts.googleapis.com/css?family=Denk+One" rel="stylesheet">
-	<title>find Pokemon meetups!</title>
+	<title>Catch a Pokémon meetup</title>
 </head>
+
 <body>
-	<h1>Find Pokemon meetups near you!</h1>
+	<header>
+		<img class="logo" src="pictures/logo.png" alt="logo" width="250px">
+		<h1>Catch a Pokémon meetup</h1>
+		<img class="ball" src="pictures/pokeball-logo.png" alt="ball" width="100px">
+	</header>
+
 	<div class="type-zip">Your zip code <input type="text" id="zip" name="zip">
-	<input type="submit" id="sub" name="sub"></div>
+	<input type="submit" id="sub" name="sub">
```

Comment:
Keep indentation uniform when possible. I would put the input within the div at least a tab inside to help make it clear what the div contains.